lib/create-add.js

```diff
@@ -31,19 +31,35 @@ function verifyArguments(name, options) {
     );
   }
 
-  if (
-    typeof options.assertMessage !== "string" ||
-    options.assertMessage.length === 0
-  ) {
-    throw new TypeError("'assertMessage' option must be a non-empty String");
+  ["assertMessage", "refuteMessage"].forEach((m) => {
+    if (!isValidMessageOption(options[m])) {
+      throw new TypeError(`'${m}' option must be a non-empty String`);
+    }
+  });
+}
+
+/**
+ * returns true when message option is considered valie,
+ * i.e. when it is a function or a non-empty string
+ *
+ * @private
+ * @param  {*}  message [description]
```

Comment:
Can we omit the `[description]` if no description is added?

Reply:
Yes